todo_app/app.py

```diff
@@ -1,45 +1,108 @@
-from flask import render_template, Flask, request, redirect, url_for
-from todo_app.data.session_items import get_item, get_items, save_item, delete_item, add_item
-
+from flask import Flask, redirect, url_for, render_template, request
+import requests
+import os
 from todo_app.flask_config import Config
 
 app = Flask(__name__)
-
 app.config.from_object(Config)
 
+class Item:
+    def __init__(self, id, title, status):
+        self.id = id
+        self.title = title
+        self.status = status
+
+API_KEY = os.environ.get("api_key")
+TOKEN = os.environ.get("token")
```

Comment:
You need to update the README.md file to explain setup requirements like these env vars and even having a Trello account.